### PR TITLE
Update to version 1.1.0

### PR DESCRIPTION
**Pull request by techrah**
Preparing for release...

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: featureselection
 Title: Feature Selection
-Version: 1.0.0.0
+Version: 1.1.0.100
```
Comment (jackyho112) [inline DESCRIPTION]: Why `100`?

Comment (techrah) [reply DESCRIPTION]: No reason in particular. Just odd to have a build number of zero.

Comment (techrah) [reply DESCRIPTION]: Shall I change it to `0`?

Comment (techrah) [reply DESCRIPTION]: Fixed

**Review by jackyho112 — APPROVED** [review]
Looks good!

**Pull request merged**